src/coprocessor/codec/data_type/vector.rs

```diff
@@ -175,6 +176,15 @@ impl VectorValue {
         Ok(())
     }
 
+    #[inline]
+    pub fn get_unchecked(&self, index: usize) -> ScalarValueRef<'_> {
```

Comment:
I wanted to avoid the `get` name because `slice::get` will return an `Option`. I wasn't aware that `slice::get_unchecked` is unsafe - -| 
Actually, the semantic I expect equals to `&vec[index]`, which will panic if index is out of range. But I cannot implement `std::ops::Index` because it must return a reference. `ScalarValueRef<'_>` is a reference itself. So...any suggestions to its name?

Reply:
How about get_scalar?

Reply:
I change it to get_scalar_ref